styles/palette.css

```diff
@@ -4,34 +4,40 @@
   --stanford-cardinal: rgb(var(--stanford-cardinal-rgb));
   --stanford-black: #2e2d29;
   --stanford-black-rgb: 46, 45, 41;
+  --stanford-90-black: #43423E;
+  --stanford-90-black-rgb: 67,66, 62;
   --stanford-80-black: #585754;
   --stanford-70-black: #6d6c69;
+  --stanford-60-black: #767674;
   --stanford-50-black: #979694;
   --stanford-40-black: #ababa9;
   --stanford-30-black: #c0c0bf;
+  --stanford-20-black: #d5d5d4;
   --stanford-10-black: #eaeaea;
   --stanford-10-black-rgb: 234, 234, 234;
 
   /* https://identity.stanford.edu/design-elements/color/accent-colors/ */
   --stanford-poppy-rgb: 233, 131, 0;
   --stanford-poppy-dark: #d1660f;
   --stanford-palo-alto-dark-rgb: 1, 66, 64;
+  --stanford-palo-alto-dark: #175e54;
   --stanford-stone-dark-rgb: 84, 73, 72;
   --stanford-illuminating-dark-rgb: 254, 197, 29;
   --stanford-fog-light: #f4f4f4;
   --stanford-sky-dark: #016895;
   --stanford-sky-dark-rgb: 1, 104, 149;
 
   /* https://identity.stanford.edu/design-elements/color/web/ */
+  --stanford-digital-blue: #006cb8;
   --stanford-digital-blue-rgb: 0, 108, 184;
+  --stanford-digital-blue-dark: #00588c;
   --stanford-digital-blue-dark-rgb: 0, 84, 143;
   --stanford-digital-green-dark-rgb: 0, 111, 84;
   --stanford-digital-red-dark-rgb: 130, 0, 0;
   --stanford-digital-green: #008566;
   --stanford-digital-red: #b1040e;
+  --stanford-digital-red-light: #e50808;
```

Comment:
Is this used by anything in the component library?  I'd prefer that if it's not used in any of our designs, we keep it out, even if it is in the current identity guide.